RELEASE.md

```diff
@@ -1,293 +0,0 @@
-# Release Workflow <a id="release-workflow"></a>
```

Comment:
I opt for having an internal guideline not per repo.

Reply:
I suggest keeping at least the parts that are special to this repo and have to be done in this repo (i.e. `internal/version.go` and `AUTHORS`) in here until such a thing exists. So that you can follow `icinga2/RELEASE.md` with the extra steps given here.

Reply:
Added a version check to GitHub actions. For the `AUTHORS` file I would just enable our Probot here.